tests/test_config.json

```diff
@@ -1,27 +1,32 @@
 {
+    "debug": false,
     "use_real_mongo": false,
     "mongo_database": "optimade",
     "mongo_uri": "localhost:27017",
     "links_collection": "links",
     "references_collection": "references",
     "structures_collection": "structures",
-    "page_limit": 500,
+    "page_limit": 20,
+    "page_limit_max": 500,
     "default_db": "test_server",
+    "base_url": null,
     "implementation": {
         "name": "Example implementation",
-        "source_url": "https://github.com/Materials-Consortia/optimade-python-tools"
+        "version": "0.7.0",
+        "source_url": "https://github.com/Materials-Consortia/optimade-python-tools",
+        "maintainer": null
```

Comment:
Does this mean one _must_ provide all these fields in the config file now?

Reply:
No, the defaults will kick in via the Pydantic Model.

Reply:
Because, I like the idea of _not_ having, e.g., `version` in the config file, and rather have it "update" automagically, when you increase the version of the server/package :)

Reply:
oh yeah, and that's a good example of just updating the defaults you want to change.

The way its implement the order of precedence is this:
1.) Any environment variable
2.) Any value in `config_file`
3.) Defaults in `ServerConfig`

Reply:
Ooohh, stuff for the README / INSTALL! :)